### PR TITLE
Expand Grief Prevention page

### PR DESCRIPTION
**Pull request by equanimityspace**
https://wiki.left4craft.org/claims/

Added more information, fixed commands, added more commands, added command section, cleaned up markdown formatting a little.

### CODE REVIEW

**Comment by equanimityspace** [comment]
Formatting looks okay now

File: docs/claims/index.md [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-|/TrustList	                          |Lists the permissions for the claim you're standing in.	
-|/SubdivideClaim                          |Switches your shovel to subdivision mode, so you can subdivide your claims.
+Left4Craft uses the Grief Prevention Plugin to allow players an opportunity to protect their land and valuables against mobs, griefing and other troublesome events.
+
+## Basics
+
+Your first claim can be started by placing a chest on the ground, which will automatically claim a 9x9 area centered on the chest. There are temporary indicators to show the borders of your land claim. You may only have one automatic chest claim at a time.
+
+To claim land manually, you will first need to aquire a golden shovel. With the shovel right click two corners of land, the claim must be a minimum of five blocks wide and use at least 100 claim blocks. You can check see the border indicators of your claim at any time by right-clicking a block inside your claim with a stick. To get the indicators to go away, right-click a block outside of your claim.
```
Comment (eartharoid) [inline docs/claims/index.md]: Perhaps mention `/kit claim`.

**Review by eartharoid — CHANGES_REQUESTED** [review]
Looks good

File: docs/claims/index.md (hunk 1 above)
Comment (equanimityspace) [reply docs/claims/index.md]: done

**Review by eartharoid — APPROVED** [review]

**Pull request merged**